package.json

```diff
@@ -12,6 +12,7 @@
     "networks-extract": "CONF_FILE=$(pwd)'/migration_conf.js'  node node_modules/@gnosis.pm/util-contracts/src/extract_network_info.js",
     "networks-inject": "CONF_FILE=$(pwd)'/migration_conf.js'  node node_modules/@gnosis.pm/util-contracts/src/inject_network_info.js",
     "networks-reset": "mkdir -p build/contracts && npx truffle networks --clean && npm run networks-inject",
+    "verify-stablex": "npx truffle run verify StablecoinConverter",
```

Comment:
ubernit: I think when running node scripts, the `node_modules/.bin/` is automatically added to the path so there is no need to use `npx`.